app.rb

```diff
@@ -0,0 +1,88 @@
+require_relative './classes/student'
+require_relative './classes/teacher'
+require_relative './association/book'
+require_relative './association/rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    list = []
+    @books.to_a.each { |item| list << "Title: \"#{item.title}\" Author #{item.author}" }
+    list.length.positive? ? list : 'No Element Found'
+  end
+
+  def list_people
+    list = []
+    if @people.length.positive?
+      @people.each do |item|
+        list << "[#{item.class}] Name: #{item.name}, ID: #{item.object_id}, Age: #{item.age}"
+      end
+    end
+
+    list.length.positive? ? list : 'No Element Found'
+  end
+
+  def list_rentals(id)
+    puts 'Rentals: '
+    @rentals.each do |item|
+      puts "Date #{item.date}, Book \"#{item.book.title}\" by #{item.person.name}" if item.person.object_id == id
+    end
+  end
```

Comment:
[REQUIRED] The list of rentals is not displayed after I enter the person's ID

Reply:
@fobadara Thank you for your comment,
Kindly recheck my app  I think you probably entered the wrong ID, and that is why you can't see any result.
Below  is a screenshot that demonstrates my list of rentals functionality
![showcase](https://user-images.githubusercontent.com/26736582/193398890-b02e662d-7452-4b4b-bc13-c92a365228a6.png)

As you can see from this screenshot you get no result when you enter a non-existing ID and once you pick  an existing ID you can see the result